Python/gc_free_threading.c

```diff
@@ -1928,7 +1928,8 @@ invoke_gc_callback(PyThreadState *tstate, const char *phase,
         info = Py_BuildValue("{sisnsn}",
```

Comment:
You need to change this line to 

```suggestion
        info = Py_BuildValue("{sisnsnsd}",
```

no?

Reply:
Nice catch, thanks. Annoying that it just ignores trailing args like that.